koa-crud/test/integration/graphql/promo-enrollment-requests.spec.ts

```diff
@@ -550,15 +550,26 @@ describe('Promo Enrollment Queries', function () {
       this.depositMockId = depositMock._id;
       this.signUpMockId = signUpMock._id;
 
+      const dateToday = new Date();
+
+      const baseDate = new Date(dateToday);
+
+      baseDate.setMinutes(dateToday.getMinutes() + 30);
+
```

Comment:
This would cause brittle test. I think the best way to control time in test is to fake it using sinon.

https://sinonjs.org/releases/v9.2.4/fake-timers/

Reply:
Noted on this